packages/components/src/popover/index.js

```diff
@@ -115,6 +115,7 @@ const Popover = (
 		__unstableSlotName = SLOT_NAME,
 		__unstableObserveElement,
 		__unstableForcePosition,
+		__unstableShift = false,
```

Comment:
It looks like switching this to default to `false` caused a small regression for the Tooltip component which relies on the shift behaviour to prevent the Tooltip from causing the editor to overflow. I've opened up a separate PR to opt-in to the `__unstableShift` for the Tooltip in https://github.com/WordPress/gutenberg/pull/41524